mobile/lib/modules/campaign/widgets/detail/campaign_info.widget.dart

```diff
@@ -44,8 +44,8 @@ class CampaignInfo extends StatelessWidget {
             ),
             OrganizationInfo(
               organization: OrganizationModel(
-                id: campaign.organizationId,
-                name: campaign.organizationName,
+                id: campaign.organizationId!,
```

Comment:
I recommend using null check and add default value for null safety `currentCampaign.organizationName ?? 'Some foo default value'`